src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Shield.shared.cs

```diff
@@ -52,16 +52,16 @@ public Color SubjectBackgroundColor
 		[EditorBrowsable(EditorBrowsableState.Never)]
 		public static readonly BindableProperty TextColorProperty =
 			BindableProperty.Create(nameof(TextColor), typeof(Color), typeof(Shield), Color.Default,
-				propertyChanged: OnTextColorChanged);
+				propertyChanged: OnStatusTextColorChanged);
 
+		[Obsolete("TextColor is obsolete. Please use StatusTextColor instead")]
+		[EditorBrowsable(EditorBrowsableState.Never)]
 		public Color TextColor
 		{
-			get => (Color)GetValue(TextColorProperty);
-			set => SetValue(TextColorProperty, value);
+			get => (Color)GetValue(StatusTextColorProperty);
```

Comment:
Isn't this change a breaking change? At least a binary one

Reply:
I discarded this code